.env

```diff
@@ -0,0 +1,2 @@
+REACT_APP_API_URL=https://fallingfruit.org/test-api/0.2
+REACT_APP_API_KEY=
```

Comment:
I don't think this .env file should be committed into the repo, only the example.env should be present

Reply:
@Qrtn tagging just to confirm

Reply:
Yes, this .env file shouldn't be committed because it's your personal env config. Thanks @laurenho025

Reply:
Remove this file, as it's a personal config.